### DISCUSSION TITLE
Selection of network card

### DISCUSSION THREAD

**Discussion by Comiter** [sanny32/OpenModSim#12] [Q&A] [answered]
I'm trying to configure the Modbus slave to be active on a specific NIC om my computer, but I'm not able to select which NIC the slave should connect to, I'm only able to select which port it should listen to.
Is there a setting in any of the configuration files, or is the IP selected  based on first NIC with connection?

**Comment by sanny32** [comment]
Hello @Comiter. I'm sorry, you can't configure specific NIC in Open ModSim utility.

**Comment by Comiter** [comment]
> Hello @Comiter. I'm sorry, you can't configure specific NIC in Open ModSim utility.

Ah, okay. So the OpenModSim is not able to reply to requests from network traffic at all? It is only designed to test applications on local computer?

**Comment by sanny32** [comment]
No, Open ModSim accept connection from any network interface.

**Reply by Naga3028-oss** [reply]
How can we configure the IP, in my case i have to vary IP range in simulator to connect multiple server. it is possible ??

**Reply by sanny32** [reply]
No, Open ModSim accept connection from any network interface.

**Comment by sanny32** [comment, marked as answer]
This functionality available from version 1.7 by selecting ip address of modbus tcp server
